### PR TITLE
Added functionality to add legal notices and cgu

### PR DESCRIPTION
**Pull request by thocle86**

### CODE REVIEW

**Review by CeliaBoiss — DISMISSED** [review]

**Review by BoulMaker — DISMISSED** [review]

File: src/Controller/LegalsTextsController.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            switch ($type) {
+                case "cgu":
+                    $legalsTexts->setType('cgu');
+                    break;
+                case "legal-notice":
+                    $legalsTexts->setType('legal-notice');
+                    break;
+            }
```
Comment (RomainClair) [inline src/Controller/LegalsTextsController.php]: You could also have a type field in the form, may be a hidden one. Dealing with the form data would be done by the form component

File: src/Entity/LegalsTexts.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+     *      minMessage = "Votre texte doit faire au moins {{ limit }} charactères",
+     *      maxMessage = "Votre texte ne peut pas faire plus de {{ limit }} charactères"
+     * )
+     */
+    private string $text;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $type;
```
Comment (RomainClair) [inline src/Entity/LegalsTexts.php]: You should verify your type, specially because it's one choice between a few number of choices.

**Review by RomainClair — CHANGES_REQUESTED** [review]
Migrations not compatible with the current dev branch

**Review by Quentin-BLANC — DISMISSED** [review]

**Review by BoulMaker — DISMISSED** [review]

**Review by RomainClair — APPROVED** [review]

**Review by RomainClair — CHANGES_REQUESTED** [review]
Migration file incompatible with current dev branch

**Review by CeliaBoiss — APPROVED** [review]

**Review by BoulMaker — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**